### PR TITLE
[Expand for PROGRAMMES-6316][fix]: Display recipes with different sizes

### PR DESCRIPTION
**Pull request by tatitati**
Reference in V2:
https://github.com/bbc/programmes/blob/develop/webapp/php/sharedmodules/programmes/views/partials/programmes/recipes.spv#L49-L52

URLs examples:
Odd amount of recipes: 
http://sandbox.bbc.co.uk/programmes/b0111461/recipes
Event amount of recipes:
http://sandbox.bbc.co.uk/programmes/b013pqnm/recipes

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: templates/recipes/share/2013incBody.html.twig [hunk 1]
```diff
@@ -4,8 +4,15 @@
     </div>
     <div class="component__body br-box-page">
         <ul class="grid-wrapper highlight-box-wrapper highlight-box-wrapper--grid@bpb1 highlight-box-wrapper--grid@bpw">
+            {%- set isOddAmountOfRecipes = (recipes|length) % 2 != 0 -%}
+
             {%- for recipe in recipes -%}
-                <li class="grid 1/2@bpb1 1/2@bpw">
+                {%- set recipeClassees = 'grid 1/2@bpb1 1/2@bpw' -%}
+                {%- if isOddAmountOfRecipes and loop.index == 1 -%}
+                    {%- set recipeClassees = 'grid 1@bpb1 1@bpw' -%}
```
Comment (votemike) [inline templates/recipes/share/2013incBody.html.twig]: Is 1@bpb1 a class? or Is it 1/1@bpb1?

File: templates/recipes/share/2013incBody.html.twig [hunk 2]
```diff
@@ -4,8 +4,15 @@
     </div>
     <div class="component__body br-box-page">
         <ul class="grid-wrapper highlight-box-wrapper highlight-box-wrapper--grid@bpb1 highlight-box-wrapper--grid@bpw">
+            {%- set isOddAmountOfRecipes = (recipes|length) % 2 != 0 -%}
+
             {%- for recipe in recipes -%}
-                <li class="grid 1/2@bpb1 1/2@bpw">
+                {%- set recipeClassees = 'grid 1/2@bpb1 1/2@bpw' -%}
```
Comment (votemike) [inline templates/recipes/share/2013incBody.html.twig]: recipeClasses not recipeClassees

File: templates/recipes/share/2013incBody.html.twig [hunk 3]
```diff
@@ -4,8 +4,15 @@
...
+                {%- if isOddAmountOfRecipes and loop.index == 1 -%}
```
Comment (votemike) [inline templates/recipes/share/2013incBody.html.twig]: Small thing. Would loop.first be clearer? Because the loop might be 0 or 1 indexed.

**Pull request merged**